### DISCUSSION TITLE
can we save the pagination setting to the chart metadata

### DISCUSSION THREAD

**Discussion by J71-haha** [apache/superset#18529] [Ideas]
The superset chart table sets the number of pagination rows to select the setting, and add the option of whether to select all

![image](https://user-images.githubusercontent.com/52438024/141405634-182b7d09-3631-44bd-9b8d-7c7f2627b188.png)
like this
![image](https://user-images.githubusercontent.com/52438024/141405677-f9e25aef-e0d3-4e99-986a-ff531c5b49b4.png)

### Expected results

what you expected to happen.

### Actual results

what actually happens.

### Environment

(please complete the following information):

- browser type and version:
- superset version: `superset version` ：1.3.2
- python version: `python --version`：3.7
- node.js version: `node -v`
- any feature flags active:

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by junlincc** [comment]
make sense. should be an easy enhancement. Contribution is welcome! we are happy to guide you through the contribution process! :)

**Comment by Carlosbogo** [comment]
Hi, I would love to help with this, but I am still getting used to working with github and I haven't worked in this repository before. Is there anyone already working on this? If not, I would like to start working on it.

**Comment by zhaoyongjie** [comment]
Hi @Carlosbogo, welcome to contribution, if you need to help, feel free ping me any time.

**Comment by Carlosbogo** [comment]
Sorry for the delay, but I'm not sure if I understand the issue correctly:
Should the chart show the total number of entries instead of just "all" or am I missing the point?
